lib/ii.c

```diff
@@ -13072,25 +13072,37 @@ grn_ii_select_data_find_phrase_product(grn_ctx *ctx,
   uint32_t phrase_group_id = data->token_info->phrase_group_id;
   phrase_group *group = &(data->phrase_groups[phrase_group_id]);
   bool skipped = false;
-  token_info *min_before = group->btree->min;
   while (true) {
     data->token_info = group->btree->min;
+    grn_posting min_posting_before = *(data->token_info->p);
     if (grn_ii_select_data_find_phrase(ctx,
                                        data,
                                        data->token_info->phrase_id,
                                        start_pos,
                                        start_n_tokens_in_phrase)) {
       skipped = true;
       bt_reorder_min(group->btree);
-      token_info *min = group->btree->min;
-      if (min == min_before) {
-        break;
-      }
-      if (min->pos == min_before->pos &&
-          min->n_tokens_in_phrase > min_before->n_tokens_in_phrase) {
+      /* This condition decision whether same phrase or not by using a posting.
```

Comment:
@yoshimotoyuk Could you review English for this comment?
(@komainu8 may need to explain this in Japanese.)